.circleci/workflows.yml

```diff
@@ -1355,6 +1364,16 @@ workflows:
             - setup
           dry_run: << pipeline.parameters.dry_run >>
           rc_requested: << pipeline.parameters.rc_requested >>
+      - build_n_push_am_ce_job:
+          context: cicd-orchestrator
+          graviteeio_version: << pipeline.parameters.graviteeio_version >>
+          dry_run: << pipeline.parameters.dry_run >>
+          tag_latest: << pipeline.parameters.tag_latest >>
+          tag_latest_support: << pipeline.parameters.tag_latest_support >>
+      - publish_am_rpms:
+          context: cicd-orchestrator
+          dry_run: << pipeline.parameters.dry_run >>
+          graviteeio_version: << pipeline.parameters.graviteeio_version >>
```

Comment:
This parameter is expected to avoid RPM generation for RC/alpha releases

`          rc_requested: << pipeline.parameters.rc_requested >>`